Dockerfile

```diff
@@ -6,15 +6,34 @@
 # Written by: Michael Pellon <m@pellon.io>
 # Updated by: Chandrapal <bnchandrapal@protonmail.com>
 # Updated by: Steve Micallef <steve@binarypool.com>
+# Updated by: Steve Bate <svc-spiderfoot@stevebate.net>
 #    -> Inspired by https://github.com/combro2k/dockerfiles/tree/master/alpine-spiderfoot
 #
 # Usage:
 #
 #   sudo docker build -t spiderfoot .
-#   sudo docker run -it -p 5001:5001 spiderfoot
+#   sudo docker run -p 5001:5001 spiderfoot
+#
+# Using Docker volume for spiderfoot data
+#
+#   sudo docker run -p 5001:5001 -v /mydir/spiderfoot:/var/lib/spiderfoot spiderfoot
+#
+# Running spiderfoot commands without web server (can optionally specify volume)
+#
+#   sudo docker run --rm spiderfoot -h
+#
+# Running spiderfoot unit tests in container
+#   
+#   sudo docker run --rm --entrypoint='/usr/bin/python3' spiderfoot -m unittest discover -s test/unit
```

Comment:
```suggestion
#   sudo docker run --rm --entrypoint='/usr/bin/env python3' spiderfoot -m unittest discover -s test/unit
```

maybe ?

Reply:
Can you elaborate on your rationale? My understanding is that this would be used in shebang lines to allow a user's PATH to specify different python3 interpreter than the system default. In this case, there's only one interpreter in the image. Also, the --entrypoint CLI arg doesn't allow passing arguments. We'd need to do something like:

`docker run --rm --entrypoint=/usr/bin/env spiderfoot python3 -m unittest discover -s test/unit`

If we leave the ENTRYPOINT directive in the Dockerfile as ["/usr/bin/python3"] (vs. my change), the advantage is this would become:

`sudo docker run --rm spiderfoot -m unittest discover -s test/unit`

at the expense of needing to specify the sf.py file for regular command line operation. For example,

`sudo docker run --rm spiderfoot sf.py -M`

to list modules. The image basically becomes a Python interpreter customized for spiderfoot.

Reply:
Fair enough